api/app/signals/apps/graphql/middleware.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib.auth.models import AnonymousUser
+from rest_framework.exceptions import AuthenticationFailed
+
+from signals.auth.backend import JWTAuthBackend
+
+
+class AuthenticationMiddleware:
```

Comment:
Left over from my experiment of needing a user to access `private` information. Not needed for now because we decided to start with exposing information that is already public available, but I did not want to throw this away because it could be helpful in the future implementation of the `private` part.

Reply:
Great, definitely leave it.